### DISCUSSION TITLE
uninstalling plexupdate?

### DISCUSSION THREAD

**Discussion by nmbrg** [tteck/Proxmox#2249] [Help Needed]
i added plexupdate on my plex lxc thru using:
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/plex.sh)"

but i have since changed my mind since im having issues with it using autoscan.
and im now trying to remove plexupdate, but havent found a way to do this.

so, im hoping i could get some help here on how to completely remove plexupdate when its installed with the help of the tteck plex script.

**Comment by tteck** [comment]
If there's a cron created in crontab (`crontab -e`), please remove it. 
`rm -rf /opt/plexupdate /etc/plexupdate.conf /etc/plexupdate.cron.conf /etc/cron.daily/plexupdate`
https://github.com/mrworf/plexupdate/issues/256

**Reply by nmbrg** [reply]
Those two last lines didn’t find any files, so I guess no cron job was set up.

so that was it? All that was needed to remove it?
